bindata/cloud-network-config-controller/controller.yaml

```diff
@@ -32,7 +52,12 @@ spec:
         image: {{.CloudNetworkConfigControllerImage}}
         imagePullPolicy: IfNotPresent
         command: ["/usr/bin/cloud-network-config-controller"]
-        args: ["-platform-type", "{{.PlatformType}}", "-platform-region", "{{.PlatformRegion}}", "-secret-name", "cloud-credentials"]
+        args: [ "-platform-type", "{{.PlatformType}}",
+                "-platform-region={{.PlatformRegion}}",
+                "-platform-api-url={{.PlatformAPIURL}}",
+                "-platform-aws-ca-override={{.PlatformAWSCAPath}}",
```

Comment:
So this is always set? Will this not fail if there is no custom CA? The other PR seems to unconditionally os.Open it and here we only set the `PlatformAWSCAPath` variable if there actually is a custom CA or did I miss something?

Reply:
good question. It will be empty string in the standard case, so no big deal; the cncc will ignore it.